pathfinding_service.rst

```diff
@@ -41,11 +41,11 @@ See `Routing Preferences`_ for information on how to configure the trade-off bet
 
 Public Interfaces
 =================
-The path finding service needs three public interfaces
+The pathfinding service provides four public interfaces
 
-* a public endpoint for path requests by Raiden nodes
-* an endpoint to get updates from blockchain events
-* an endpoint to get updates about current balances and fees
+* A public REST endpoint for requests by Raiden nodes
+* An interface to get updates from the blockchain
+* An endpoint to receive broadcasted updates about channel balances and mediation fees
 
```

Comment:
here you list just three public interfaces, right?

Reply:
Nice catch. So I thought as well that this is talking about the REST endpoints, but this is about interfaces in general. But I guess we can remove this as it doesn't give any valuable information.